pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -42,6 +42,8 @@ type sarApprover struct {
 	recognizers []csrRecognizer
 }
 
+// NewCSRApprovingController creates a new certificate.CertificateController object
+// that handles approving certificate signing requests
```

Comment:
Add periods to all comments https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences